examples/sri-1/components/workgroup/workflow/workflow.ts

```diff
@@ -0,0 +1,21 @@
+import { IWorkflow } from './workflow.interface';
+import { Workstep } from '../workstep/workstep';
+
+export class Workflow implements IWorkflow {
+  id: string; //[R234] A workflow with more than one workstep MUST have a unique identifier within a BPI.
```

Comment:
We either add references to standard for every property or nowhere. I vote for the second option. It would be too much noise in the code. Test cases will describe the standard requirements.

Reply:
references removed form components.